javascript/js/main.js

```diff
@@ -1,58 +1,203 @@
-console.log("_______ les-26 _________");
-
-// 1. Створити конструктор Accumulator.
-// Даний конструктор має приймати число,
-// яке буде початковим значенням. Клас має
-// реалізувати методи increment, при виклику якого
-// передане значення має збільшуватись на 1, та decrement,
-// при виклиці якого значення має зменшуватись.
-
-// 2. Створити конструктор CancelableAccumulator, 
-// який буде наслідувати Accumulator та додатково 
-// буде реалізовувати метод clear, при виклику якого 
-// значення збереженого числа має стати таким, 
-// яким його передали в конструкторі.
-
-function Accumulator(volum) {
-  this.volum = volum;
-  this.decrement = function () {
-    this.volum--;
-  };
+console.log("_______ les-27 _________");
+
+let arr = [21, 130, -99, 3, 28, 11, 10, [15, 9], "text", 12];
+
+// check the area on "string" and creat new arr "notString"
+function checkArrea(a) {
+    let checkArr = [];
+    let i = 0;
+    while (i < a.length) {
+        if (typeof (a[i]) === "number") checkArr.push(a[i]);
```

Comment:
don't forget that `NaN` also return type number so check also for `!isNaN(a[i])`